src/Offer/ReadModel/Permission/Doctrine/DBALRepository.php

```diff
@@ -85,4 +85,13 @@ public function markOfferEditableByUser(StringLiteral $eventId, StringLiteral $u
             // permission record is already in place.
         }
     }
+
+    public function markOfferEditableByNewUser(StringLiteral $eventId, StringLiteral $userId): void
```

Comment:
Something very minor, but I wonder if "NewOwner" would be a better name than "NewUser"?

Reply:
You are right, in context as the interface of `DBALRepository` the `newUser` makes more sense. Keep the method name as is.